### DISCUSSION TITLE
Is the doc about cordova prerequisited updated?

### DISCUSSION THREAD

**Discussion by realtebo** [quasarframework/quasar#15175] [CLI - Capacitor / Cordova modes]
> As per December 2019 Cordova requires android-28 (Android 9.0 - Pie) so be sure to include it.

Quoted from: https://quasar.dev/quasar-cli-vite/developing-cordova-apps/preparation#windows

Which is the actual need of cordova?

**Comment by metalsadman** [comment]
when quoting include the links where the source came from. P.R are welcome.

**Reply by realtebo** [reply]
Updated the question. I cannot make a PR because I'ne no the right answer
